cmd/export_accounts.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/stellar/stellar-etl/internal/input"
+	"github.com/stellar/stellar-etl/internal/transform"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+var accountsCmd = &cobra.Command{
+	Use:   "export_accounts",
+	Short: "Exports the account data.",
+	Long:  `Exports historical account data within the specified range to an output file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		startNum, endNum, limit, path, useStdOut := utils.MustBasicFlags(cmd.Flags(), cmdLogger)
+
+		var outFile *os.File
+		if !useStdOut {
+			outFile = mustOutFile(path)
+		}
+
+		accounts, err := input.GetAccounts(startNum, endNum, limit)
+		if err != nil {
+			cmdLogger.Fatal("could not read accounts: ", err)
+		}
+
+		for _, acc := range accounts {
+			transformed, err := transform.TransformAccount(acc)
+			if err != nil {
+				cmdLogger.Fatal("could not transform account", err)
+			}
+
+			marshalled, err := json.Marshal(transformed)
+			if err != nil {
+				cmdLogger.Fatal("could not json encode account", err)
+			}
+
+			if !useStdOut {
+				outFile.Write(marshalled)
+				outFile.WriteString("\n")
+			} else {
+				fmt.Println(string(marshalled))
+			}
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(accountsCmd)
+	utils.AddBasicFlags("accounts", accountsCmd.Flags())
+	ledgersCmd.MarkFlagRequired("end-ledger")
```

Comment:
Not sure I follow this. Why are you modifying `ledgersCmd`? Or should this be `accountsCmd`?

Reply:
Good point. That should be accountsCmd.